pkg/test/framework/components/echo/instances.go

```diff
@@ -213,3 +177,94 @@ func (i Instances) Contains(instances ...Instance) bool {
 	})
 	return len(matches) > 0
 }
+
+// Services is a set of Instances that share the same FQDN. While an Instance contains
```

Comment:
Ah did we change `Instance` to have one deployment per cluster? If so, is the need for `Services` to collect all `Instance`s that are subsets of the same service (fqdn)? The docs weren't making this clear ... maybe another pass through the description of `Instance`, `Service`, and `RunToN` would be helpful? Concrete examples of subsetting would be good here.

Reply:
Before I update the docs in-repo, let's assert here:

* `Instance` can contain multiple deployments (subsets) of 1 service
* `Instances` contains an arbitrary set of `Instance`
* `Service` is an `Instances` where every underlying `Instance` shares the FQDN (the default behavior of calls to the FQDN would be loadbalanced across all Workloads in all Instances). 
* `Services` is an arbitrary set of `Service`.